css/style.css

```diff
@@ -296,29 +845,248 @@ li a{
   display: inline-block;
 }
 
+.sort-head{
+  margin-right: 230px;
+}
+
+.sort-items input{
+  display: none;
+}
+
+.sort-items label{
+  margin-right: 25px;
+}
+
+.sort-items input[type="radio"]:checked + label{
+  color: #000000;
+}
+
+.sort-items input[type="radio"] + label{
+  color: #b2b2b2;
+}
+
+.sort-increase input{
+  display: none;
+}
+
+.down-arrow:checked + label{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-down-checked.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.down-arrow + label{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-down-unchecked.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.down-arrow + label:hover{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-down-hover.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.up-arrow:checked + label{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-up-checked.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.up-arrow + label{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-up-unchecked.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.up-arrow + label:hover{
+  content: "";
+  width: 9px;
+  height: 9px;
+  background-image: url(../img/sort-up-hover.png);
+  background-repeat: no-repeat;
+  margin-right: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
 .result-templates{
   float: right;
   width: 760px;
   padding-top: 55px;
 }
 
 .template-demo{
-  outline: 2px dotted red;
   float: left;
   width: 360px;
   height: 616px;
   margin-bottom: 40px;
+  overflow: hidden;
+  position: relative;
+}
+
+.template-demo:hover{
+  box-shadow: 0 0 35px -10px #000000;
+}
+
+.template-demo:hover .browser{
+  background-image: url(../img/browser-dark.png);
+}
+
+.template-image{
+  width: 360px;
+  height: 576px;
+}
+
+.pink{
+  background-image: url(../img/img-pink.jpg);
+}
+
+.sedona{
+  background-image: url(../img/img-sedona.jpg);
+}
+
+.barbershop{
+  background-image: url(../img/img-barbershop.jpg);
+}
+
+.mishka{
+  background-image: url(../img/img-mishka.jpg);
+}
+
+.aplus{
+  background-image: url(../img/img-aplus.jpg);
+}
+
+.kvast{
+  background-image: url(../img/img-kvast.jpg);
+}
+
+.browser{
+  height: 40px;
+  background-image: url(../img/browser-light.png);
+}
+
+.template-demo-checked{
+  position: absolute;
+  width: 360px;
+  height: 231px;
+  background-color: #eeeeee;
+  bottom: 0;
+  text-align: center;
+  transform: translateY(100%);
```

Comment:
Transform property should be used with vendor prefixes to provide support for older browsers (e.g. Safari).